### ISSUE TITLE
I have a general problem with widget, expecially IMAGE widget and SITE LOGO widget,

### ISSUE THREAD

**Issue by ansianinja** [elementor/elementor#14708]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

 
Always when i insert or select an image widget or a site logo widget it can't be edited like in that photo.

<img src="http://lucat12.sg-host.com/wp-content/uploads/2021/04/Schermata-2021-04-26-alle-10.44.36.png" alt="Elementor Problem" width="100%">

## Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [X] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by fleks** [comment]
Switch Wordpress language to English (United States)
More here: https://flextensions.com/6099/problemi-di-elementor-con-lultimo-aggiornamento-2/elementor-it/problemi/2021/

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.2.2

Duplicate of #14642

Feel free to update

Thanks!
